plugins/jira/reference/create-story.md

```
@@ -0,0 +1,111 @@
+### Test-Based
+```plaintext
+- Test that <criteria>
+```
+
+### Verification-Based
+```plaintext
+- Verify that when <a role> does <some action> they get <this result>
+```
+
+### Given-When-Then (BDD)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)